### PR TITLE
[IMPROVEMENT] About activity to fragment

### PR DESCRIPTION
**Pull request by divyanshub024**
@RocketChat/android

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/about/ui/AboutFragment.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import chat.rocket.android.BuildConfig
+import chat.rocket.android.R
+import chat.rocket.android.main.ui.MainActivity
+import kotlinx.android.synthetic.main.app_bar.*
+import kotlinx.android.synthetic.main.fragment_about.*
+
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/about/ui/AboutFragment.kt:13]: Remove this blank line.

File: app/src/main/java/chat/rocket/android/about/ui/AboutFragment.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class AboutFragment : Fragment() {
+
+    companion object {
+        fun newInstance() = AboutFragment()
+    }
+    override fun onCreateView(inflater: LayoutInflater,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/about/ui/AboutFragment.kt]: Add a blank line here.

File: app/src/main/res/layout/fragment_about.xml [hunk 3]
```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    tools:context="chat.rocket.android.about.ui.AboutFragment">
```
Comment (philipbrito) [inline app/src/main/res/layout/fragment_about.xml:6]: 👍

**Review by philipbrito — CHANGES_REQUESTED** [review]
One less TODO, thank you! \o/
Just some needs before to go.

**Comment by divyanshub024** [comment]
@filipedelimabrito all changes are made.

**Review by philipbrito — APPROVED** [review]
Thank you very much @divyanshub024!

**Pull request merged**